test/e2e/pages/vl-proza-message.page.js

```diff
@@ -1,5 +1,7 @@
 const VlProzaMessage = require('../components/vl-proza-message');
+const { VlAlert } = require('vl-ui-alert').Test;
```

Comment:
Moeten we ook geen vl-ui-toaster importeren?

Reply:
Aan de vl-ui-toaster ben ik deze morgen aan begonnen

Reply:
en nen button, icon en typography?
ik zou dezelfde depedencies hier verwachten dan dat er bij de dependencies in de package.json staan ...

Reply:
Button trekt icon binnen dus die hoeft ook niet als extra dependency hier te staan. Heb wel een extra test toegevoegd hiervoor.